tests/std/tests/VSO_0000000_instantiate_containers/test.cpp

```diff
@@ -46,6 +46,20 @@ using namespace std;
 
 int main() {} // COMPILE-ONLY
 
+// FUNCTION TEMPLATE For_each_tuple_element
+template <class Tpl, class Fx, size_t... Indices>

```

Comment:
This test file uses `typename` for template parameters. (This is one of the things where we allow tests to follow different conventions than the product - but we still require consistency within each file.)

```suggestion
template <typename Tpl, typename Fx, size_t... Indices>
```